### ISSUE TITLE
Diseño del Index

### ISSUE THREAD

**Issue by v1siond** [WoWDeliveryApp/deliveryApp#3]
La index representa la cara de la aplicación, es la primero que el usuario verá y por tanto, es lo primero en la lista de diseños.

Requisitos:

- Diseño Minimalista
- Header Menu -> Inicio - ¿Cómo Funciona? - Registrarte - Ingresar 
- Footer -> mapa del sitio, terminos y condiciones/políticas de privacidad, copyright, redes sociales
- Trabajar en dimensiones 1920 x 1080, para efectos de calidad a la hora de implementar el diseño.

Se debe manejar un diseño para contacto, hay 2 maneras planteadas, una donde el contacto esté al final del index (antes del footer) y otra donde sea una página aparte, lo dejo a tu criterio.

**Comment by Deathsona** [comment]
https://drive.google.com/file/d/0B3oFmf_ocZwxZFJqeS13OFVwWjg/view

**Comment by Deathsona** [comment]
https://drive.google.com/drive/folders/0B3oFmf_ocZwxUzdjX1o0TXhJU00

**Comment by Deathsona** [comment]
https://drive.google.com/file/d/0B3oFmf_ocZwxMC1DOGh4cERBRTA/view?usp=sharing

**Comment by v1siond** [comment]
Está brutal, la única pega es el texto, hay que encontrar una manera de que sea mas legible. También creo que el tamaño es muy grande.

**Comment by Deathsona** [comment]
https://drive.google.com/file/d/0B3oFmf_ocZwxd1RnRnNrYVNGaEk/view?usp=sharing

**Comment by Deathsona** [comment]
https://drive.google.com/open?id=0B3oFmf_ocZwxWlpuSDRwQnRCbHc

**Comment by loldlm1** [comment]
Es muy depinga el index, aunque tal vez podriamos probar con otros Fonts para las letras, aqui dejo un ejemplo de los mejores fonts usados en el 2016 por diseñadores orientados a webs http://www.webdesignerdepot.com/2016/06/101-best-free-fonts-of-2016-so-far/

**Comment by v1siond** [comment]
Estoy de acuerdo, dejemos un sólo mensaje a modo eslogan, delivery al alcance de tu mano me parece perfecto, la legibilidad del eslogan es la cosa

**Comment by Deathsona** [comment]
https://drive.google.com/open?id=0B3oFmf_ocZwxd3FxcG8zWmRTclU
https://drive.google.com/open?id=0B3oFmf_ocZwxcFA0ZktKaHY1Xzg
https://drive.google.com/open?id=0B3oFmf_ocZwxN09BYks5aWF2OTQ
https://drive.google.com/open?id=0B3oFmf_ocZwxZkFqUm9ubHdIaWs

**Comment by v1siond** [comment]
Para mi, el 4 está perfecto. Agrega el footer y con eso está listo
